src/components/Box/index.js

```diff
@@ -14,12 +14,16 @@ function Box(props) {
 
   return (
     <ConfigConsumer>
-      {({ styleWith = () => className }) => {
+      {({ getDeclarationsForClasses, processDeclarations }) => {
         if (className) {
-          remainingProps.className = `${styleWith(className, !!important)}`.trim()
+          const { declarations, ignoredClasses } = getDeclarationsForClasses(className)
+          const processedDeclarations = processDeclarations(declarations, (declaration) => css`${declaration}`)
+        
+          remainingProps.css = processedDeclarations
+          remainingProps.className = ignoredClasses.join(" ").trim()
```

Comment:
What's `trim` do here? I assumed all of the `ignoredClasses` would already be trimmed?

Reply:
You're right, this was a leftover fragment.